linux/secureboot_enable_disable/secureboot_enable_disable.yml

```diff
@@ -35,28 +40,33 @@
             secureboot_enabled_pass: false
             secureboot_disabled_pass: false
 
-        - include_tasks: ../utils/shutdown.yml
+        - name: "Shutdown guest OS"
+          include_tasks: ../utils/shutdown.yml
 
-        # Enable secureboot when secureboot is not enabled
-        - include_tasks: ../../common/vm_set_boot_options.yml
+        - name: "Enable secure boot for VM"
+          include_tasks: ../../common/vm_set_boot_options.yml
           vars:
             secure_boot_enabled_set: true
           when: not vm_secureboot_enabled
 
-        - include_tasks: ../../common/vm_set_power_state.yml
+        - name: "Power on VM"
+          include_tasks: ../../common/vm_set_power_state.yml
           vars:
             vm_power_state_set: 'powered-on'
 
-        - include_tasks: ../../common/update_inventory.yml
+        - name: "Update inventory"
+          include_tasks: ../../common/update_inventory.yml
 
-        - include_tasks: ../../common/vm_wait_log_msg.yml
+        - name: "Check VM image is approved in vmware.log when secure boot is enabled"
+          include_tasks: ../../common/vm_wait_log_msg.yml
           vars:
             vm_wait_log_name: "vmware.log"
             vm_wait_log_msg: "Image APPROVED"
             vm_wait_log_retries: 1
             vm_wait_log_ignore_errors: true
 
-        - include_tasks: ../utils/get_system_firmware.yml
+        - name: "Get system firmware in guest OS"
```

Comment:
No need to get the firmware here, and condition on line 76 can also be removed, since if firmware==bios, the test case has already been skipped at the beginning.